package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "eslint-plugin-ubervu",
+  "version": "0.0.0",
+  "description": "ESLint rules applicable to the uberVU org",
+  "keywords": [
+    "eslint",
+    "eslintplugin",
+    "eslint-plugin"
+  ],
+  "author": "uberVU",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint --ext .js --ext .jsx --format './node_modules/eslint-friendly-formatter' .",
+    "test": "mocha"
+  },
+  "dependencies": {
+    "requireindex": "~1.1.0"
```

Comment:
We don't need this anymore. It was used in the initial index.js generated by https://www.npmjs.com/package/generator-eslint but I changed the structure a bit.